src/python/startup.cpp

```diff
@@ -0,0 +1,32 @@
+// BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/master/LICENSE
+
+#include <iostream>
+#include <string>
+
+#include "awkward/python/startup.h"
+
+// namespace ak = awkward;
+
+const std::string StartupLibraryPathCallback::library_path() const {
+  std::string library_path = ("/");
```

Comment:
This should return some invalid path, like an empty string to be caught by the utility function that handles this output, so that we don't end up debugging issues like, "Why does it think `libawkward-cuda-kernels.so` is in the root of my filesystem?"

Reply:
Oh, I see the issue here. I'll make this change.